### ISSUE TITLE
test: Enable xdist

### ISSUE THREAD

**Issue by ghost** [freedomofpress/securedrop#2755]
# Feature request

## Description

Run tests in parallel (using xdist) to speed up the global runtime. 

@tmc has a draft implementation at #2571 

## User Stories

As a SecureDrop contributor I would like tests to be as quick as possible.

**Comment by redshiftzero** [comment]
Related: #2535

**Comment by heartsucker** [comment]
Note, this depends on #2877 in all likelihood.

**Comment by redshiftzero** [comment]
this is a duplicate of #2535, closing
